cmd/ls-main.go

```diff
@@ -102,6 +106,12 @@ EXAMPLES:
 
   9. List all objects on mybucket, summarize the number of objects and total size.
      {{.Prompt}} {{.HelpName}} --summarize s3/mybucket/
+
+  10. List all objects on mybucket, for all storageClass
+     {{.Prompt}} {{.HelpName}} --storage-classes '*' s3/mybucket
```

Comment:
```suggestion
     {{.Prompt}} {{.HelpName}} --storage-classes '*' s3/mybucket
```

This is default - so don't know if this needs an example @Willena

Reply:
Fixed !